Makefile

```diff
@@ -256,98 +319,126 @@ else
 endif
 
 #
-# Rule for building dist-tarballs, SPEC files, RPMs, debian collateral, deb's.
+# Rules for building distro packages.
 #
 
-dist:
-	$(Q)eval `$(GIT_ID) .` && \
-	tardir=cri-resource-manager-$$gitversion; \
-	tarball=cri-resource-manager-$$gitversion.tar; \
-	echo "Creating $$tarball.$(TAR_SUFFIX)..."; \
-	rm -fr $$tardir $$tarball* && \
+ifneq ($(DISTRO_ID),fedora)
```

Comment:
centos?

Reply:
We need that only for fedora, which is nowadays more of a rolling release distro IMO than separate versions. We try to handle it as a single entity. For Centos we have version-specific dockerfiles in place.